Header header
int32 MODE_FLAG
geometry_msgs/Pose ee_goal
sensor_msgs/JointState joint_goal
